libnavigation-core/src/main/java/com/mapbox/navigation/core/internal/utils/RouteOptionsEx.kt

```diff
@@ -0,0 +1,17 @@
+package com.mapbox.navigation.core.internal.utils
+
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.navigation.navigator.internal.MapboxNativeNavigatorImpl
+
+/**
+ * Check if uuid is valid:
+ * - [RouteOptions] is not **null**;
+ * - uuid is not empty;
+ * - uuid is not equal to [MapboxNativeNavigatorImpl.OFFLINE_UUID].
+ */
+fun RouteOptions?.isValidUuid(): Boolean = run {
```

Comment:
Why a `run` instead of just a method block?

Reply:
agree, re-make with just `this`